sound/soc/sof/topology.c

```diff
@@ -1821,59 +1858,30 @@ static int sof_process_load(struct snd_soc_component *scomp, int index,
 			       ARRAY_SIZE(comp_tokens), private->array,
 			       le32_to_cpu(private->size));
 	if (ret != 0) {
-		dev_err(sdev->dev, "error: parse process.cfg tokens failed %d\n",
+		dev_err(sdev->dev, "error: parse process tokens failed %d\n",
```

Comment:
separate patch?

Reply:
well, yeah I'm just fixing things what check-patch wants me to do, I should then do intermediate commits without hooks. These are old stuff that checkpatch doesn't like. Anyway I can put all comment changes to separate patch.